### ISSUE TITLE
[Quest} Old Hillsbrad Foothills quest bug (Not Thrall bug)

### ISSUE THREAD

**Issue by Pitypalaty** [WoWManiaUK/Redemption#1452]
After i plant the bombs and escort Thrall and do everything what i need to do, i just can't turn in the quest at Erozion (or what's his name). Drop and redo not working either. It's really annoying since i need that to be able to move on to Black Morass dungeon to do Master Of Elixirs quest.... Please, reply.

**Comment by tvshadowz** [comment]
Heya Pityoakaty! Thanks for the report, I'll check this right away. 

In future could you make sure you link quests and npcs with their ids  or their wowhead links. For further information on how to report bugs have a look here: http://www.wow-mania.com/forums/topic/71

Thanks for the report!

**Comment by tvshadowz** [comment]
This worked as intended in the live client.

**Comment by fvicaria** [comment]
I will re-check this from scratch when I get back. If you can describe exactly at what point during the quest line it fails and how it will help. Links to NPC and to quests also help.

**Comment by babbadook** [comment]
i still have the same problem as pitypalaty did it also many times but ezerion don't has a question mark so he don't teleport me in to time to medivh to fix the broken key that's needed to get into black morass . i do this qst chain to obtain the masters key needed to get into karazhan .

**Comment by Flux13** [comment]
I ran into this bug today while soloing the dungeon for the elixir master quest line (http://www.wowhead.com/quest=29599/escape-from-durnholde). Erozion, the quest giver (http://www.wowhead.com/npc=18723/erozion), appeared outside the building that Thrall runs out of before the final boss encounter had started, and disappeared soon after the boss was dead. There wasn't an option to turn in the quest when I talked to him during this time even though he had the quest complete question mark above his head. After he disappeared I ran back to the start of the instance where he first gives you the incendiary bombs. He still had the question mark above his head, and I still wasn't able to complete the quest.
